airflow/sensors/base.py

```diff
@@ -84,30 +82,6 @@ def __bool__(self) -> bool:
         return self.is_done
 
 
-@provide_session
```

Comment:
Oh, cool... you even found dead code while implementing the patch! Thanks for the cleaning!

Reply:
@jscheffl Thanks for support!